pkg/collector/listeners/docker.go

```diff
@@ -468,3 +483,19 @@ func findKubernetesInLabels(labels map[string]string) bool {
 	}
 	return false
 }
+
+// Rancher 1.x containers don't have docker networks as the orchestrator provides
+// its own CNI. The IP is stored in the `io.rancher.container.ip` label.
+func findRancherIPInLabels(labels map[string]string) (string, bool) {
```

Comment:
I'm wondering if we should create a `rancher.go`/`rancher_test.go` files with this function so we can avoid having too much rancher related code in the "generic" docker listener, WDYT?

Reply:
I split the `docker_kubelet.go` file because of the build tag. We could create a `DockerRancherService` but as it's a future-legacy system, I don't think that's gonna change much in the future. It might be more of a burden (update interfaces and debug issues) than an help.

Reply:
I was more thinking a simple go file holding the function rather than a full service but I get your point

Reply:
moved it in the `util/docker` package